news/admin.py

```diff
@@ -1,12 +1,17 @@
 from django.contrib import admin
 
 from news import forms
-from news.models import Post
+from news.models import Post, Category
+
+
+class CategoryAdmin(admin.ModelAdmin):
+    list_display = ('name',)
+    search_fields = ('name',)
 
 
 class PostAdmin(admin.ModelAdmin):
 
-    list_display = ('title', 'author', 'created_time', 'modified_time', 'published', 'albins_angels')
+    list_display = ('title', 'author', 'category', 'created_time', 'modified_time', 'published', 'albins_angels')
```

Comment:
Albin's angles will have it's own logic that checks if the category is Albin's angels in the future, right?

Reply:
Yes, I just want to deploy in 2 parts so that all the data is in the correct format once we do the frontend change